docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -70,13 +70,17 @@ outside of those test subdirectories.  They will also not reconfigure or bounce
 
    Use the ``--docker-no-pull`` option to avoid pulling the latest container image. This is required when using custom local images that are not available for download.
 
-Run as follows for all POSIX platform tests executed by our CI system::
+Run as follows for all POSIX platform tests executed by our CI system in a fedora32 docker container::
 
-    ansible-test integration --docker fedora29 -v shippable/
+    ansible-test integration shippable/ --docker fedora32 --docker-no-pull
```

Comment:
@Andersson007 While you are here, can you please include this fix https://github.com/ansible/ansible/issues/27446

Reply:
@Akasurde I think the issues relates to unit tests, doesn't it?
Anyway docs/docsite/rst/dev_guide/testing_units.rst has the reference to https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_data/requirements/units.txt which contains `pytest-xdist`, so, i believe that the issue is irrelevant and can be closed

Reply:
OK. Thanks.

Reply:
Yw!